elasticsearch/linux_distributions/opendistro-tar-install.sh

```diff
@@ -21,6 +21,15 @@ bash $ES_HOME/plugins/opendistro_security/tools/install_demo_configuration.sh -y
 ##Perf Plugin
 chmod 755 $ES_HOME/plugins/opendistro_performance_analyzer/pa_bin/performance-analyzer-agent
 chmod -R 755 /dev/shm
+chmod a+rw /tmp
+chown -R elasticsearch $ES_HOME/performance-analyzer-rca
+chown elasticsearch $ES_HOME/data/rca_enabled.conf
```

Comment:
what's the reason for this change? for tarball distribution, any uid can be used for service startup. And the elasticsearch doesn't exist probably.

Reply:
changed to chmod with 755